src/app/Instance/BridgeOverview/Components/BODashboardTableView.tsx

```diff
@@ -38,7 +39,10 @@ export interface BOTableItem {
 
 interface BODashboardTableViewProps {
   itemsList: BOTableItem[] | undefined;
+
   name: string;
+  onEditProcessor: (processorId: string) => void;
+  deleteProcessor: (processorId: string, processorName: string) => void;
```

Comment:
This is a generic table re-used for all the resources inside `BridgeOverview`, not just processors but also sources and sinks.

Choosing processor specific prop names looks not right. Those should be called `onEditItem` and `onDeleteItem`, or something similar.